### ISSUE TITLE
13->16: Unrecognized attribute(s) 'availability_zone_hints'

### ISSUE THREAD

**Issue by jistr** [os-migrate/os-migrate#205]
Error importing routers into OSP 16 when they were exported from OSP 13.

```
     Traceback (most recent call last):
      File "/root/.ansible/tmp/ansible-tmp-1591633747.1084464-1378-104171785932711/AnsiballZ_import_router.py", line 102, in <module>
        _ansiballz_main()
      File "/root/.ansible/tmp/ansible-tmp-1591633747.1084464-1378-104171785932711/AnsiballZ_import_router.py", line 94, in _ansiballz_main
        invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
      File "/root/.ansible/tmp/ansible-tmp-1591633747.1084464-1378-104171785932711/AnsiballZ_import_router.py", line 40, in invoke_module
        runpy.run_module(mod_name='ansible_collections.os_migrate.os_migrate.plugins.modules.import_router', init_globals=None, run_name='__main__', alter_sys=True)
      File "/usr/lib64/python3.7/runpy.py", line 205, in run_module
        return _run_module_code(code, init_globals, run_name, mod_spec)
      File "/usr/lib64/python3.7/runpy.py", line 96, in _run_module_code
        mod_name, mod_spec, pkg_name, script_name)
      File "/usr/lib64/python3.7/runpy.py", line 85, in _run_code
        exec(code, run_globals)
      File "/tmp/ansible_os_migrate.os_migrate.import_router_payload_7y12yupm/ansible_os_migrate.os_migrate.import_router_payload.zip/ansible_collections/os_migrate/os_migrate/plugins/modules/import_router.py", line 117, in <module>
      File "/tmp/ansible_os_migrate.os_migrate.import_router_payload_7y12yupm/ansible_os_migrate.os_migrate.import_router_payload.zip/ansible_collections/os_migrate/os_migrate/plugins/modules/import_router.py", line 113, in main
      File "/tmp/ansible_os_migrate.os_migrate.import_router_payload_7y12yupm/ansible_os_migrate.os_migrate.import_router_payload.zip/ansible_collections/os_migrate/os_migrate/plugins/modules/import_router.py", line 107, in run_module
      File "/tmp/ansible_os_migrate.os_migrate.import_router_payload_7y12yupm/ansible_os_migrate.os_migrate.import_router_payload.zip/ansible_collections/os_migrate/os_migrate/plugins/module_utils/resource.py", line 201, in create_or_update
      File "/tmp/ansible_os_migrate.os_migrate.import_router_payload_7y12yupm/ansible_os_migrate.os_migrate.import_router_payload.zip/ansible_collections/os_migrate/os_migrate/plugins/module_utils/router.py", line 53, in _create_sdk_res
      File "/usr/lib/python3.7/site-packages/openstack/network/v2/_proxy.py", line 2471, in create_router
        return self._create(_router.Router, **attrs)
      File "/usr/lib/python3.7/site-packages/openstack/proxy.py", line 224, in _create
        return res.create(self, base_path=base_path)
      File "/usr/lib/python3.7/site-packages/openstack/resource.py", line 1116, in create
        self._translate_response(response)
      File "/usr/lib/python3.7/site-packages/openstack/resource.py", line 962, in _translate_response
        exceptions.raise_from_response(response, error_message=error_message)
      File "/usr/lib/python3.7/site-packages/openstack/exceptions.py", line 229, in raise_from_response
        http_status=http_status, request_id=request_id
    openstack.exceptions.BadRequestException: BadRequestException: 400: Client Error for url: https://controller16.v2v.bos.redhat.com:13696/v2.0/routers, Unrecognized attribute(s) 'availability_zone_hints'
```

The YAML files need to be edited and the `availability_zone_hints` property needs to be removed from them before importing.

**Comment by jistr** [comment]
We may want to deal with this in some automated-but-generic-and-clean manner :) ...

**Comment by jistr** [comment]
Fixed in #335
